README.md

```diff
@@ -0,0 +1,159 @@
+# next-experiments
+
+Next-experiments is an attempt to add A/B testing infrastructure to the Next.js
+
+**Warning: The project is in the alpha stage, consider reviewing source code of the project before using it in the production**
+
+## Getting Started
+
+### Demo
+
+Check out samples folder to get a basic concepts of next-experiments
+
+### How to add A/B testing to your next.js site
+
+#### Prerequsistes
```

Comment:
Can we put this above `### Demo`, as `### Quickstart`? Like this:

### Quickstart
> ⚠️ *For the time being, you must install our next.js fork ([link](https://github.com/Sheertex/next.js/blob/ABTesting/contributing.md#running-your-own-app-with-locally-compiled-version-of-nextjs)), which adds `permuteStaticPaths` support

Install library:
```bash
yarn add next-experiments
```
Change your `next.config.js` to add [...]:
```js
// example
```

Once you implement the `permuteStaticPaths` and `getStaticProps` hooks, next-experiments will be able to statically compile separate files for every possible `<Experiment>` + `<Variant>` combination on your page:
```js
// page example, with page just rendering the `<TestExperiment />` directly
```

This works for both static and dynamic page paths. When you next statically compile the site, variants will be rendered to your `out` directory:
```bash
next build
next export
```

### Advanced Usage ...

Reply:
Done